src/Modal/Modal.test.jsx

```diff
@@ -134,65 +115,62 @@ describe('<Modal />', () => {
   });
   describe('invalid keystrokes do nothing', () => {
     beforeEach(() => {
-      wrapper = mount(
-        <Modal
-          {...defaultProps}
-        />,
-      );
+      wrapper = mount(<Modal
+        {...defaultProps}
+      />);
     });
 
     it('does nothing on invalid keystroke q', () => {
-      const buttons = wrapper.find('button');
+      let buttons = wrapper.find('button');
 
-      expect(buttons.at(0).matchesElement(document.activeElement)).toEqual(true);
+      expect(buttons.at(0).html()).toEqual(document.activeElement.outerHTML);
       modalOpen(true, wrapper);
+      // buttons.at(0).prop('keyDown')({ key: 'q' });
```

Comment:
Extra comment? Can we kill this line?

Reply:
errr this might've been an accidental comment...

![alt-text](https://media.giphy.com/media/BxWTWalKTUAdq/giphy.gif)